src/__tests__/utils.test.ts

```diff
@@ -14,13 +13,15 @@ import {
   pick,
 } from '../utils';
 
+import { parseISO } from 'date-fns';
```

Comment:
Can you move this import to line 2 just for convenience and code organization, please?

Reply:
Import line removed